src/file_readers/common/sql_reader.rs

```diff
@@ -12,21 +12,27 @@ pub struct SqlReader {
 }
 
 impl SqlReader {
-    fn read_column_from_table<
-        T: rusqlite::types::FromSql + std::default::Default,
-    >(
+    fn read_column_from_table<T: rusqlite::types::FromSql + Default>(
```

Comment:
Very minimal change, check if if it formatted right by running `cargo fmt`. Normally thse kind sof changes are done automaticaly. (I evne set up my editor (VSCode with rust-analyzer extension) to run `cargo fmt` automatically on saving)